src/main/webapp/app/config/axios-interceptor.ts

```diff
@@ -12,8 +12,23 @@ const setupAxiosInterceptors = onUnauthenticated => {
   const onResponseSuccess = response => response;
   const onResponseError = err => {
     const status = err.status || (err.response ? err.response.status : 0);
+    const url = err.config.url;
+
+    // Ignore 401 errors from the root URL
```

Comment:
rozumiem że póki co to taki workaround? myślę że ten problem leży gdzieś w logice, musimy go tylko kiedyś przy okazji odkryć ^^